crates/re_types_builder/src/codegen/cpp/mod.rs

```diff
@@ -26,6 +26,8 @@ use self::method::{Method, MethodDeclaration};
 
 use super::common::ExampleInfo;
 
+type Result<T = (), E = anyhow::Error> = std::result::Result<T, E>;
```

Comment:
Why not `anyhow::Result` then?

Reply:
Future-proof if we want to change the error to `CodegenError` or whatever 🤷